JapaneseFilter/sections/specific.txt

```diff
@@ -182,6 +182,8 @@ punyu.com##.box_title_pr + .rank_table
 punyu.com##.box_title[style="width:330px; margin-top:10px;"]
 punyu.com##.tblmain[cellspacing="2"][cellpadding="0"]
 punyu.com##div[id^=dugaflvplayer-]
+iphone-mania.jp###jack
```

Comment:
I suggest to change it to `iphone-mania.jp##div[id^="jack"]`. If you can't make a change, please let me know.

Reply:
That causes FP.

Reply:
Hm, I guess you meant incorrect blocking. What about this `iphone-mania.jp###jack_wrapper > div[id^="jack"]` one?

Reply:
Looks okay, will change.

Reply:
Done.